spec/features/activities/project_attributes_activity_spec.rb

```diff
@@ -88,7 +89,7 @@
       # own fields
       expect(page).to have_selector('li', text: "Name changed from #{previous_project_attributes['name']} to #{project.name}")
       expect(page).to have_selector('li', text: 'Description set (Details)')
-      expect(page).to have_selector('li', text: 'Visibility set to public')
```

Comment:
That's nice, when inserting the new assertion, you respected the same order in the assignments and the assertions.

It looks like the visibility change text assertion was removed. Can you restore it please?